test/domain/notification/removedPost.spec.ts

```diff
@@ -0,0 +1,26 @@
+
+import { beforeEach, describe, expect, it } from 'vitest';
+
+import getByPostId from '^/domain/notification/getByPostId';
+import removedPost from '^/domain/notification/notify/removedPost';
+import { DATABASES, VALUES } from './fixtures';
```

Comment:
There should be an empty line before this import. Those above are coming from the `domain`, and this one from `./`